helpdesk_stock/views/helpdesk_views.xml

```diff
@@ -56,12 +56,13 @@
                             <field name="direction" required="1"/>
                             <field name="route_id"
                                    options="{'no_create': True}"
-                                   groups="stock.group_stock_multi_locations"/>
+                                   groups="stock.group_stock_multi_locations"
+                                   invisible="1"/>
```

Comment:
No blocking comment. Invisible="1" and groups both attributes don't survive together.

Reply:
@osi-scampbell can you UI check this?